test/com/google/enterprise/adaptor/TestHelper.java

```diff
@@ -118,4 +109,17 @@ public static void initSSLKeystores() {
         TestHelper.class.getResource("/test-cacerts.jks").getPath());
     System.setProperty("javax.net.ssl.trustStorePassword", "changeit");
   }
+
+  /** Gets a proxy for the given class that does nothing. */
+  public static <T> T newProxyInstance(Class<T> clazz) {
```

Comment:
Would you accept doNothingProxy? I'd like to keep Proxy in the names of things that use Proxy.